packages/use-contractkit/src/contract-kit-provider.tsx

```diff
@@ -86,12 +110,23 @@ export const ContractKitProvider: React.FC<ContractKitProviderProps> = ({
     [isMountedRef]
   );
 
+  const methods = useContractKitMethods(state, dispatch);
+
+  useEffect(() => {
+    if (CONNECTOR_TYPES[state.connector.type] !== UnauthenticatedConnector) {
+      void methods.initConnector(state.connector);
```

Comment:
Recommend adding a .catch here to print error and make linter happy

Reply:
This is actually a good point and maybe we should do more here. If the connector fails to initialize on mount then we should probably clear it out of state.

Reply:
Agreed!